pyinfra/facts/server.py

```diff
@@ -452,6 +455,11 @@ def process(self, output):
                     if group_name and group_name != group:
                         groups.append(group_name)
 
+                # Parse lastlog info
+                # lastlog output varies, which is why I use regex to match login time
+                login = re.search(rex, lastlog)
+                login_time = login.group() if login else '**Never logged in**'
```

Comment:
```suggestion
                login_time = login.group() if login else None
```

I think a `None` for never logged in here is more Python-y and easier to compare to.

Reply:
I went with '''**Never logged in**''' because that's the output from lastlog.  However, I agree that using `None` in this case is probably better.